apps/hash-frontend/src/pages/login.page.tsx

```diff
@@ -1,7 +1,7 @@
 import { TextField } from "@hashintel/design-system";
```

Comment:
I think we should rename this route to `/signin` now that we've updated the links to refer to it as such

Reply:
Have addressed in https://github.com/hashintel/hash/pull/4009/commits/1a82571defd8f9c8f1152b6952cf2bde27a8d36e